scripts/deploying.js

```diff
@@ -1,3 +1,5 @@
+/* globals $,document, jsyaml */
```

Comment:
Out of curiosity, what's the purpose?

Reply:
makes eslint not gripe about undefined so it can focus on my real problems.